lcls-twincat-motion/Library/POUs/Motion/States/Examples/FB_PositionState1D_InOut.TcPOU

```diff
@@ -47,9 +47,9 @@ IF stOut.sName = '' THEN
     stOut.sName := 'OUT';
 END_IF
 
-// Assemble the states array, matching the enum values (IN is 1, OUT is 2)
-astPositionState[1] := stIn;
-astPositionState[2] := stOut;
+// Assemble the states array, matching the enum values (OUT is 1, IN is 2)
+astPositionState[1] := stOut;
```

Comment:
self-review thought before I leave today: this should probably look more like this:
```
astPositionState[E_EpicsInOut.OUT] := stOut
```